app/views/components/TaxReturnLink.scala.html

```diff
@@ -25,6 +25,6 @@
 @returnItem = @{"return-item-" + (index+1)}
 
 <li id=@returnItem class="govuk-list">
- <p class="govuk-heading-m">@message
- <br><a class="govuk-body" aria-label="@message" href="@call">@linkText</a></p>
+ <p class="govuk-heading-m govuk-!-margin-bottom-1">@message
+ <p><a class="govuk-body" aria-label="@message" href="@call">@linkText</a></p>
```

Comment:
this doesnt look like valid html 🤔 

`<li><p><p><a></a></p></li>`

Reply:
yes, you right. I will look into it